engine/src/Options/Classes/Particles/Size/Size.ts

```diff
@@ -1,3 +1,5 @@
+import { EasingType } from "../../../../exports";
```

Comment:
Don't reference the root `export` file, use the right file from the `Enums` folder. The root files are for library usage only. They shouldn't be used inside the library code, this will prevent circular references.